INSTALL.md

```diff
@@ -1666,6 +1666,13 @@ most UNIX/Linux systems), and Windows threads.  No other threading models are
 supported.  If your platform does not provide pthreads or Windows threads then
 you should use `Configure` with the `no-threads` option.
 
+For pthreads, all locks are non-recursive. In addition, in a debug build,
+the mutex attribute `PTHREAD_MUTEX_ERRORCHECK` is used. If this is not
+available on your platform, you might have to add
```

Comment:
Do we have reason to believe that such platforms exist?

Reply:
I am pretty sure that our reference platforms all have `PTHREAD_MUTEX_ERRORCHECK`  I am less sure of other platforms such as VMS or Tandem non-stop.  If nothing else, it avoids a question that might come up. :)